app/src/main/kotlin/at/bitfire/davdroid/repository/DavSyncStatsRepository.kt

```diff
@@ -61,11 +61,14 @@ class DavSyncStatsRepository @Inject constructor(
         val packageManager = context.packageManager
         val packageName = packageManager.resolveContentProvider(authority, 0)?.packageName ?: authority
         return try {
-            val appInfo = packageManager.getPackageInfo(packageName, 0).applicationInfo
+            val appInfo = packageManager.getPackageInfo(packageName, 0).applicationInfo!!
             packageManager.getApplicationLabel(appInfo).toString()
         } catch (e: PackageManager.NameNotFoundException) {
             logger.warning("Application name not found for authority: $authority")
             authority
+        } catch (_: NullPointerException) {
+            logger.warning("Package name ($packageName) not found for authority: $authority")
+            authority
```

Comment:
Throwing an NPE and then catching it is more difficult then just to check for null. We can just check for null and return `authority` in this case.